### ISSUE TITLE
OpenAI Assistants API integration

### ISSUE THREAD

**Issue by sthitap0** [pipecat-ai/pipecat#321]
Hello!
Appreciate your work with Pipecat.
I had an openAI assistant with file search and tool call. Is there a way to use it with Pipecat using the OpenAI Assistants api?

**Comment by geekofycoder** [comment]
Yes the assistants API is sounding good.This is in beta testing but surely will be a boom for this domain.Any ideas @TomTom101 @chadbailey59 @aconchillo

**Comment by lcareto** [comment]
any update on that?

**Comment by markbackman** [comment]
Closing as this is not in scope for Pipecat.
